dev/com.ibm.ws.security.social/resources/com/ibm/ws/security/social/resources/SocialMessages.nlsprops

```diff
@@ -95,7 +95,7 @@ KUBERNETES_USER_API_RESPONSE_DEFAULT_USER_ATTR_NOT_FOUND.explanation=The user AP
 KUBERNETES_USER_API_RESPONSE_DEFAULT_USER_ATTR_NOT_FOUND.useraction=Verify that the configuration attribute that is specified in the message is configured correctly. Ensure that the value of the configuration attribute corresponds to the entry in the user API response that should be used as the username.
 
 # 0=Exception message
-ERROR_INTROSPECTING_SERVICE_ACCOUNT=CWWKS5382E: The response from the service account user API cannot be processed: {0}
+ERROR_INTROSPECTING_SERVICE_ACCOUNT=CWWKS5382E: The response from the user API cannot be processed: {0}
```

Comment:
Per IBM style, variable should precede descriptor

In this message, does {0} represent the response itself or the API?

Reply:
response

Reply:
per IBM style, it should read

The {0} response from the user API cannot be processed.

However, if the API response is expected to be a long string or similar, making the message illegible, it can be

The following response from the user API cannot be processed: {0}

Reply:
response can be very long and some times with an error exception included. I think it is better to do "The following response ........"